_posts/de/newsletters/2022-11-02-newsletter.md

```diff
@@ -0,0 +1,277 @@
+  **sicherheitskritische** Veröffentlichungen, die Fehlerbehebungen für ein
+  Problem bei der Verarbeitung neuer Blöcke enthalten. Alle Benutzer sollten ein
+  Upgrade durchführen.
+
+- [Bitcoin Core 24.0 RC2][] ist ein Release Candidate für die nächste Version
+  der am weitesten verbreiteten Full-Node-Implementierung des Netzwerks. Eine
+  [Anleitung zum Testen][bcc testing] ist verfügbar.
+
+  **Warnung:** Dieser Release Candidate enthält die Konfigurationsoption
+  `mempoolfullrbf`, von der mehrere Protokoll- und Anwendungsentwickler glauben,
+  dass sie zu Problemen für Händlerdienste führen könnte; wie in den Newslettern
+  [#222][news222 rbf] and [#223][news223 rbf] beschrieben. Optech fordert alle
+  Dienste auf, die davon betroffen sein könnten, den RC zu evaluieren und sich
+  an der öffentlichen Diskussion zu beteiligen.
+
+## Nennenswerte Code- und Dokumentationsänderungen
+
+*Erwähnenswerte Änderungen diese Woche in [Bitcoin Core][bitcoin core repo],
+[Core Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], und [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23927][] schränkt `getblockfrompeer` auf beschnittenen (pruned)
+  Knoten auf Blockhöhen unterhalb des aktuellen Synchronisationsfortschritts des
+  Knotens ein. Dies verhindert, dass ein Footgun entsteht, der zukünftige Blöcke
+  abruft und damit die Blockdateien des Knotens für das Pruning ungeeignet macht.
+
+  Bitcoin Core speichert Blöcke in Dateien von etwa 130 MB in der Reihenfolge,
+  in der sie empfangen werden. Beim Pruning werden ganze Blockdateien verworfen,
+  aber keine Datei, die einen Block enthält, der nicht durch die Synchronisation
+  verarbeitet wurde. Die Kombination aus einer kleinen Datenmenge und der
+  wiederholten Verwendung des `getblockfrompeer`-RPC könnte dazu führen, dass
+  mehrere Blockdateien nicht für das Pruning in Frage kommen und dass ein
+  beschnittener Knoten seine Datenmenge überschreitet.
+
+- [Bitcoin Core #25957][] verbessert die Leistung von Rescans für
+  Deskriptor-Wallets, indem der [Blockfilter-Index][topic compact block filters]
+  (falls aktiviert) verwendet wird, um Blöcke zu überspringen, die keine für das
+  Wallet relevanten UTXOs ausgeben oder erzeugen.
+
+- [Bitcoin Core #23578][] nutzt [HWI][topic hwi] und die kürzlich eingefügte
+  Unterstützung für [BIP371][] (siehe [Newsletter #207][news207 bc22558]), um
+  externe Signierunterstützung für [Taproot][topic taproot] Schlüsselpfadausgaben
+  zu ermöglichen.
+
+- [Core Lightning #5646][] aktualisiert die experimentelle Implementierung von
+  [Angeboten][topic offers], um [x-only öffentliche Schlüssel][news72 xonly] zu
+  entfernen (stattdessen werden [komprimierte Pubkeys][compressed pubkeys]
```

Comment:
Unclear what the colloquial abbreviation for _öffentliche Schlüssel_ is. Pubkeys should work though.